src/test/unit/marker-data.test.js

```diff
@@ -499,52 +499,112 @@ describe('filterRawMarkerTableToRange', () => {
       [
         'Load 1',
         0,
-        { type: 'Network', status: 'STATUS_START', startTime: 0, endTime: 1 },
+        {
+          type: 'Network',
+          id: 0x0000000100000000,
```

Comment:
Yes, I think I should've done that originally since the load number is dependent on the id.

Also yes, apart from the test where the process id matters.